src/components/Tag/index.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { FC } from "react";
+
+import { TagWrapper, SubtractComponent } from "./styledComponents";
+import { TagNameProps } from "./interface";
+
+const Tag: FC<TagNameProps> = ({ text, removeTag }) => {
+  return (
+    <TagWrapper
+      flexDirection="row"
+      justifyContent="space-between"
+      alignItems="center"
+    >
```

Comment:
We don't need to have it customisable from the outside - it never changes in design, please put it in styles.